test/e2e/apimachinery/webhook.go

```diff
@@ -141,6 +144,12 @@ var _ = SIGDescribe("AdmissionWebhook", func() {
 		testMutatingPodWebhook(f)
 	})
 
+	It("Should not be able to prevent deleting validating-webhook-configurations", func() {
```

Comment:
This should say "...validating-webhook-configurations or mutating-webhook-configurations", because the test is for both.